### PR TITLE
Replaced original template for a default font and a root display to …

### PR DESCRIPTION
**Pull request by ferdeleong**
…allow scroll down

### CODE REVIEW

**Comment by ferdeleong** [comment]
To do:
-Couldn't blur only the background images
-Finished product week 1: https://mdeleongomez-sps-summer22.appspot.com/

**Comment by acheng1** [comment]
Hi Fernanda,

When I go to [https://mdeleongomez-sps-summer22.appspot.com/](https://mdeleongomez-sps-summer22.appspot.com/), I see 4 columns with images in the background, but no text unless I scroll to the bottom.  Did you want the text to appear over the background images?

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -7,13 +7,44 @@
...
+              </div>
+          </div>
+          <div class="content-section">
+              <h1>About</h1>
+              <h1>Projects</h1>
+              <p>Developed a decentralized voting system over the Ethereum blockchain using smart contracts in Solidity. Microservices architecture built with Node.js, Docker, and IBM Cloud.</p>
+              <p>Created a Chrome extension for speed reading utilizing facial recognition to control speed, and play/stop the session. The extension was built with React and Javascript. The ML models use ML5 library to have them web based to ensure fast feedback.</p>
+              <p>Built a web page to facilitate communication between foreigners and foreign students to find roommates with similar lifestyles using web technologies such as HTML, CSS and Javascript and Reacts as a framework. </p>
+              <h1>Gallery</h1>
+              <p>I enjoy photographing architecture because it's the combination between math and art. It’s amazing how it reflects the lifestyle and ideologies of the society at the time of construction, so everytime you see a building you are looking directly at a time capsule. </p>
```
Comment (acheng1) [inline portfolio/src/main/webapp/index.html]: nit: Some of these lines seem over 80 characters long. Try to keep the lines length under 80 characters per https://developers.google.com/style/html-formatting#line-length (Apply throughout the rest of the file).

**Review by acheng1 — COMMENTED** [review]
One thing I forgot to mention is that we should use 2 spaces for indentation not 4 (see https://google.github.io/styleguide/htmlcssguide.html#Indentation)

**Comment by ferdeleong** [comment]
> Hi Fernanda,
> 
> When I go to https://mdeleongomez-sps-summer22.appspot.com/, I see 4 columns with images in the background, but no text unless I scroll to the bottom. Did you want the text to appear over the background images?

No, those images are like a header. I'm thinking that if you click on the section you want to see it scrolls down to show you the information, I don't know if I'm properly explaining myself

**Comment by ferdeleong** [comment]
> One thing I forgot to mention is that we should use 2 spaces for indentation not 4 (see https://google.github.io/styleguide/htmlcssguide.html#Indentation)

Done

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -12,6 +12,83 @@
...
+      quote: `
+        You can’t keep growing and working on yourself without accepting where you are right now.
+        Acceptance is your key to freedom.
+      `,
+      description: [
+        `
+          I believe that my education in BS Architecture and developed creativeness 
+          combined with Computer Science give me an especially great advantage when 
+          approaching problems or challenges that require creative solutions or to 
+          think out of the box, so I can bring new and fresh ideas to the table
```
Comment (acheng1) [inline portfolio/src/main/webapp/script.js:26]: nitnitnit: missing period?

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -12,6 +12,83 @@
...
+        `,
+        `
+          What I love about CS is that it gives you the tools to explore and 
+          materialize the vast universe of ideas and thoughts that humans create.
+          I build for those who can’t and need us. I build to promote autonomous 
+          human development, social cohesion and citizen integration through retribution.
+        `,
+        `
+          Im looking forward to create an app to facilitate communication between
```
Comment (acheng1) [inline portfolio/src/main/webapp/script.js:35]: nitnitnit: "I'm" instead of Im

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -12,6 +12,83 @@
...
+        `
+          What I love about CS is that it gives you the tools to explore and 
+          materialize the vast universe of ideas and thoughts that humans create.
+          I build for those who can’t and need us. I build to promote autonomous 
+          human development, social cohesion and citizen integration through retribution.
+        `,
+        `
+          Im looking forward to create an app to facilitate communication between 
+          foreigners and foreign students to find roommates based on hobbies,
+          life style, habits, etc, and create meaningful connections
```
Comment (acheng1) [inline portfolio/src/main/webapp/script.js:37]: nitnitnit: missing period

**Review by acheng1 — APPROVED** [review]

**Comment by acheng1** [comment]
> > Hi Fernanda,
> > When I go to https://mdeleongomez-sps-summer22.appspot.com/, I see 4 columns with images in the background, but no text unless I scroll to the bottom. Did you want the text to appear over the background images?
> 
> No, those images are like a header. I'm thinking that if you click on the section you want to see it scrolls down to show you the information, I don't know if I'm properly explaining myself

Ah ok, sounds reasonable if these were clickable....

**Pull request merged**